test/onnx/test_utility_funs.py

```diff
@@ -635,6 +635,27 @@ def forward(self, x):
         iter = graph.nodes()
         assert next(iter).kind() == "prim::ListConstruct"
 
+    def test_custom_layer_tuple(self):
+        class CustomFunction(torch.autograd.Function):
+            @staticmethod
+            def symbolic(g, input):
```

Comment:
How is this symbolic used here?

Reply:
Used here https://github.com/pytorch/pytorch/blob/7cdf786a07b2ca434983a0a508e2e42c75a4697d/torch/csrc/jit/passes/onnx.cpp#L309